cloud-agent/service/server/src/main/scala/org/hyperledger/identus/pollux/credentialdefinition/controller/CredentialDefinitionController.scala

```diff
@@ -43,14 +43,9 @@ trait CredentialDefinitionController {
 }
 
 object CredentialDefinitionController {
```

Comment:
This companion object and its inner members are obsolete. As explained, there is an implicit conversion available in `ErrorResponse`.

Reply:
fixed